contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,408 @@
+> 
+> Due to the dangers, we don't recommend you merge users frequently, but rather as a one-off for recovering from implementation problems. We also encourage you to consider whether merging users is strictly necessary for your data integrity, or a 'nice to have'.
+
+Merging users is done by sending a `$merge_dangerously` event:
+
+<MultiLanguage selector="tabs">
+
+```node
+client.capture({
+    distinctId: 'distinct_id_of_user_to_merge_into',
+    event: '$merge_dangerously',
+    properties: {
+        alias: 'distinct_id_of_user_to_be_merged',
+    },
+})
+```
+
+```py
+posthog.capture('distinct_id_of_user_to_merge_into', 
+  '$merge_dangerously', 
+  {'alias': 'distinct_id_of_user_to_be_merged'})
+```
+
+```go
+client.Enqueue(posthog.Capture{
+  DistinctId: "distinct_id_of_user_to_merge_into",
+  Event:      "$merge_dangerously",
+  Properties: posthog.NewProperties().
+    Set("alias", "distinct_id_of_user_to_be_merged"),
+})
+```
+
+```ruby
+posthog.capture({
+    distinct_id: 'distinct_id_of_user_to_merge_into',
+    event: '$merge_dangerously',
+    properties: {
+        alias: 'distinct_id_of_user_to_be_merged'
+    }
+})
+```
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "properties": {
+      "alias": "distinct_id_of_user_to_be_merged"
+    },
+    "timestamp": "2020-08-16 09:03:11.913767",
+    "distinct_id": "distinct_id_of_user_to_merge_into",
+    "event": "$merge_dangerously"
+}' https://app.posthog.com/capture/
+```
+
+</MultiLanguage>
+
+#### How are properties managed when merging users?
+
+
+When a `User B` is merged into another `User A`, all the properties of the `User B` are added to `User A`. If there is a conflict, the properties of `User A` are prioritized over `User B`. For example:
+```js
+client.identify({
+    distinctId: 'user_A',
+    properties: {
+        location: 'London',
+        name: 'User A',
+    },
+})
+
+client.identify({
+    distinctId: 'user_B',
+    properties: {
+        name: 'User B',
+        location: 'Rome',
+        phone: '0800-POSTHOG',
+    },
+})
+
+posthog.alias(
+  'user_A', // distinct_id
+  'user_B'  // alias_id
+)
+
+/* User B has merged into User A. User A will now have properties:
+{ 
+  name: 'User A',
+  location: 'London', 
+  phone: '0800-POSTHOG', 
+}
+*/
+```
+
+### How to split a merged user back into separate users
+
+If you've accidentally linked an incorrect anonymous ID with a specific distinct ID, or you've made a mistake when merging users, it's possible to split their combined user back into separate users. You can do this in the PostHog app by navigating to the user you'd like split, and then clicking "Split IDs" in the top right corner. 
+
+> **Warning:** This may change the numbers in your charts, even historically. We also cannot guarantee that historical data will be assigned to the correct user after the split. 
```

Comment:
@tiina303 anything worth adding/changing about this warning

Reply:
I'd just say it this way - note that it's not something that's used frequently - this way we are not limited in any way in the future.
```suggestion
> **Warning:** Going forward the distinctIDs are treated as separate users, but what happens to historical data is undefined (the distinctIDs could be split up to separate users or kept together or partially split up).
```

Reply:
```suggestion
> **Warning:** This will treat the distinct IDs as separate users for future events. However, there is no guarantee as to how past events will be treated – they may be be considered separate users, or be considered a single user for some or all events.
```